hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -504,6 +504,7 @@ class Phase(IntEnum):
     reuse = 1
     generate = 2
     shrink = 3
+    target = 4
```

Comment:
What's the argument for the order here?  Since it's internal, I'd insert target before shrink.